saleor/core/utils/random_data.py

```diff
@@ -1561,11 +1561,16 @@ def get_image(image_dir, image_name):
     return File(open(img_path, "rb"), name=image_name)
 
 
-def create_checkout_with_preorders():
+def prep_checkout():
```

Comment:
Is `prep` mean `prepare`? If yes, I would recommend to use full word instead of shortcut (the function name is short, so I think there is no need to shortcut words). 
Also, `prep_checkout` doesn't prepare a `Checkout` but `CheckoutInfo`. Lets add correct typing or rename a function

Reply:
What about this? If you don't agree, would be good to place a comment as I don't know if you miss this comment acidentialy 😄

Reply:
cc @SzymJ

Reply:
yesterday I removed some change by accident sorry for that I was sure is done :/ But its now!